src/modm/platform/extint/rp/module.lb

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2022, Nikolay Semenov
+#
+# This file is part of the modm project.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# -----------------------------------------------------------------------------
+
+def init(module):
+    module.name = ":platform:irq"
```

Comment:
Hm, the reference manual isn't very helpful in naming this module, it just calls the GPIO interrupt section "2.19.3. Interrupts" (not even "External Interrupts").
I think `:platform:irq` is perhaps too generic and easy to confuse with NVIC functionality. I would recommend `:platform:extint`, which aligns well with the SAM and STM32 naming conventions.